wp-content/plugins/core/src/Theme/Resources/Editor_Styles.php

```diff
@@ -33,4 +32,45 @@ public function visual_editor_body_class( $settings ) {
 		return $settings;
 
 	}
+
+	/**
+	 * Filter the styles sent to the MCE editor.
+	 *
+	 * We don't want to apply the full block editor styles, so
+	 * this will remove those and replace them with the CSS
+	 * specific to the MCE editor
+	 *
+	 * @param array $styles
+	 *
+	 * @return array
+	 * @filter editor_stylesheets
+	 */
+	public function visual_editor_styles( $styles ) {
+		$theme_uri = get_theme_file_uri( $this->theme_stylesheet_path() );
+		$styles    = array_diff( $styles, [ $theme_uri ] );
+		$styles[]  = get_theme_file_uri( $this->mce_stylesheet_path() );
+
+		return $styles;
+	}
+
+	/**
+	 * @return string The relative path to the theme stylesheet
+	 */
+	private function theme_stylesheet_path(): string {
+		return 'assets/css/dist/theme/master' . $this->file_suffix();
+	}
+
+	/**
+	 * @return string The relative path to the MCE stylesheet
+	 */
+	private function mce_stylesheet_path(): string {
+		return 'assets/css/dist/admin/editor-style' . $this->file_suffix();
```

Comment:
I still feel like i should rename this to tinymce-style as thats what it applies to, i can do it after merge in next work, cool? Then we can also break up more with the block styles etc as we go. "Editor style" is now confusing.